static/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -279,25 +287,30 @@ class AbstractIntegrationDetailedView<
     const tags = this.cleanTags();
 
     return (
-      <Flex>
-        {this.renderIntegrationIcon()}
-        <NameContainer>
-          <Flex>
-            <Name>{this.integrationName}</Name>
-            <StatusWrapper>
-              {this.installationStatus && (
-                <IntegrationStatus status={this.installationStatus} />
-              )}
-            </StatusWrapper>
-          </Flex>
-          <Flex>
-            {tags.map(feature => (
-              <StyledTag key={feature}>{startCase(feature)}</StyledTag>
-            ))}
-          </Flex>
-        </NameContainer>
-        {this.renderAddInstallButton()}
-      </Flex>
+      <TopSectionWrapper>
+        <Flex>
```

Comment:
Can you kill this `<Flex>` thing and make `TopSectionWrapper` into like display grid or something?

Reply:
@evanpurkhiser I got rid of this `Flex` without even making it a grid

Reply:
@evanpurkhiser Had to revert this change because it was looking weird while the page was loading. Tried using a grid but I can't get it to look right except wrapping the items on the left and right respectively into containers.

Reply:
👍